agent/tool-scripts/prometheus-metrics

```diff
@@ -191,7 +191,7 @@ case "$mode" in
 			debug_log "[$script_name] key is not defined for $host, assuming that the endpoint doesn't need authentication"
 			key="None"
 		fi
-		tool_cmd='GOPATH=$pbench_install_dir/.go PATH=$PATH:$GOPATH/bin $script_path/datalog/$tool-datalog $host $interval $tool_log $port $cert $selfsigned $key'
+		tool_cmd='PYTHONUNBUFFERED=True GOPATH=$pbench_install_dir/.go PATH=$PATH:$GOPATH/bin $script_path/datalog/$tool-datalog $host $interval $tool_log $port $cert $selfsigned $key'
```

Comment:
Maybe add the same comment as in `unittests` here to explain why `PYTHONUNBUFFERED` is needed?

Reply:
Added a comment, and updated the commit message to record the explicit fix.